python/ql/lib/semmle/python/filters/GeneratedCode.qll

```diff
@@ -32,18 +32,26 @@ class GenericGeneratedFile extends GeneratedFile {
   override string getTool() { lax_generated_by(this, result) or strict_generated_by(this, result) }
 }
 
+pragma[nomagic]
+private int minStmtLine(File file) {
+  result =
+    min(int line |
+      line = any(Stmt s | s.getLocation().getFile() = file).getLocation().getStartLine()
+    )
+}
+
+pragma[nomagic]
+private predicate isBeforeCode(Comment c, File f) {
+  f = c.getLocation().getFile() and
+  minStmtLine(f) < c.getLocation().getStartLine()
+}
```

Comment:
I know that you're just following the naming scheme set out in the file already, but isn't this predicate actually expressing that the comment `c` appears _after_ code?

Reply:
I'm very confused. I'm sure I saw a commit that fixed this, but it appears to be unfixed. :thinking:

Reply:
You definitely saw that, I'm not sure why that disappeared in [the rebase](https://github.com/github/codeql/compare/1ee4ba21f80d65a4a4192e12968df32570c7e699..1847a5713b0690fa8d39b171ed0971c44e2e6705).   

I reintroduced the fix, and also updated the expected output for the location test (that has an extra result due to the Django models getting imported).